modules/ROOT/pages/tutorial/access-control.adoc

```diff
@@ -83,10 +82,10 @@ This same database would be used by a number of different users, each with diffe
 
 Unlike applications which often require users to be modeled within the application itself, databases provide user management resources such as roles and privileges.
 This allows users to be created entirely within the database security model, a strategy that allows the separation of access to the data and the data itself.
-For more information, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/[Cypher Manual -> Access control].
+For more information, see xref:authentication-authorization/index.adoc[].
```

Comment:
More empty `[]` (I'm assuming all of these comments will have the same answer and if they don't need updating will be easily resolved, but if they need updates it felt nice to point them out when I see them)